src/commands/compile.ts

```diff
@@ -86,33 +86,20 @@ const command: Command = {
 						const embed = new MessageEmbed()
 							.setTitle("Compile Result")
 							.setFooter(`Compiled with: ${compiler}`);
+
 						if (err) {
 							embed.setColor("#BB3333");
-							embed.setDescription("Compilation failed: Errors present.");
+							embed.setDescription("Compilation failed: errors present.");
 							embed.addField("Error", err.message, true);
 						} else {
 							if (res.compiler_error) {
 								embed.setColor("#D95B18");
-								embed.setDescription("Compilation failed: Compiler errors present.");
+								embed.setDescription("Compilation failed: compiler errors present.");
 								embed.addField("Compiler Error", res.compiler_error, false);
```

Comment:
The `compiler_output` (and thus `compiler_message`, which combines `compiler_output` and `compiler_error`) is unnecessary, only compiler errors need to be shown. Alternatively, the entire compiler message can be shown too if needed by changing `res.compiler_error` to `res.compiler_message`.